doc/source/NewSiteConfigs.rst

```diff
@@ -454,30 +454,44 @@ It is recommended to increase the stacksize limit by using ``ulimit -S -s unlimi
 7. Set default compiler and MPI library (make sure to use the correct ``gcc`` version for your system and the desired ``openmpi`` version)
 
 .. code-block:: console
+
    # Check your gcc version then add it to your site compiler config.
    gcc --version
    spack config add "packages:all:compiler:[gcc@YOUR-VERSION]"
-   
+
    # Example for Red Hat 8 following the above instructions
    spack config add "packages:all:providers:mpi:[openmpi@4.1.5]"
-   
+
    # Example for Ubuntu 20.04 or 22.04 following the above instructions
    spack config add "packages:all:providers:mpi:[mpich@4.1.1]"
 
-8. If you have manually installed lmod, you will need to update the site module configuration to use lmod instead of tcl. Skip this step if you followed the Ubuntu or Red Hat instructions above.
+.. warning::
+   On some systems, the default compiler (e.g., ``gcc`` on Ubuntu 20) may not get used by spack if a newer version is found. Compare your entry to the output of the concretization step later and adjust the entry, if necessary.
+
+8. Set a few more package variants and versions to avoid linker errors and duplicate packages being built (for both Red Hat and Ubuntu):
+
+.. code-block:: console
+
+   spack config add "packages:fontconfig:variants:+pic"
```

Comment:
Note. Some/all of this can probably go into the common packages.yaml file, but I didn't have time to test this everywhere - decided to play safe.